network-uri-json.cabal

```diff
@@ -75,7 +75,7 @@ test-suite network-uri-json-tests
       aeson             >= 0.8 && < 1.5
     , base              >= 4.6 && < 4.14
     , hspec             >= 2.4 && < 2.8
-    , network-arbitrary >= 0.3 && < 0.5
+    , network-arbitrary >= 0.3 && <= 0.6
```

Comment:
For consistency use < 0.7.  That way we include any minor and patch level (non-API breaking) changes automatically.